vms/vms/utils.py

```diff
@@ -11,9 +11,10 @@
 
 def check_correct_volunteer(func):
     @wraps(func)
-    def wrapped_view(request, volunteer_id):
+    def wrapped_view(request, **kwargs):
```

Comment:
@prachi1210 @naman1901 @pavankm This is the fix needed for https://github.com/systers/vms/issues/725

Reply:
Will it not create merge issues later if both you and @anjali-dhanuka make changes in separate PRs?

Reply:
No, actually she is not doing this issue she did the other one.